document/CoreFormat.mdk

```diff
@@ -390,7 +390,8 @@ the post-processing tool should not alter any part of the `tool` object.
 {
     "name": "CodeScanner",                                       # see \(&sect;[#tool-name]\)
     "fullName": "CodeScanner 2.0, Developer Preview (en-US)",    # see \(&sect;[#tool-fullName]\)
-    "version": "2.0.0-beta",                                     # see \(&sect;[#tool-version]\)
+    "semanticVersion": "2.0.0-beta.1",                           # see \(&sect;[#tool-semanticVersion]\)
+    "version": "2.0 Beta 1 x86 (en-US)                           # see \(&sect;[#tool-version]\)
```

Comment:
After some research on this, changed to use a "real" example of non-numeric versioning: Apple's major-minor-bug-stage-prerelease schema, _e.g._, `1.1.2b12`, mapping to `1.1.2-beta.12`.